docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx

```diff
@@ -226,16 +245,25 @@ struct FloatingParticipantView: View {
 
     var participant: CallParticipant?
     var size: CGSize = .init(width: 120, height: 120)
+    var onChangeTrackVisibility: (CallParticipant?, Bool) -> Void
 
     var body: some View {
         if let participant = participant {
             VStack {
                 HStack {
                     Spacer()
 
-                    VideoRendererView(id: participant.id, size: size) { videoRenderer in
-                        videoRenderer.handleViewRendering(for: participant, onTrackSizeUpdate: { _, _ in })
-                    }
+                    VideoCallParticipantView(
```

Comment:
Do we need the logic for the floating view? This participant is not in the scroll list, so maybe we can leave the previous way as an example.